reggie/configs/data/washington.yaml

```diff
@@ -225,3 +285,39 @@ party_codes:
 
 num_party_codes:
   Not_Provided: NP
+
```

Comment:
if the "status_codes" above that Delaney defined here don't actually exist in the data (and I thought you told me they didn't?), you should remove these from the yaml, b/c otherwise it'll confuse people later